tasks/update-package-versions

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+
+const child = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+const version = process.argv[2]
+if (!version) {
+  console.error(
+    'You must supply a version.\nUsage ./update-package-versions [version]'
+  )
+  process.exit(1)
+}
+
+/**
+ * Iterates over Redwood dependencies in package.json files and updates the version.
+ */
+const updatePackageVersion = (pkgPath, version) => {
+  const pkg = JSON.parse(
+    fs.readFileSync(path.join(pkgPath, 'package.json'), 'utf-8')
+  )
+
+  if (pkg.dependencies) {
+    for (const depName of Object.keys(pkg.dependencies).filter((x) =>
+      x.startsWith('@redwoodjs/')
+    )) {
+      console.log(` - ${depName}: ${pkg.dependencies[depName]} => ^${version}`)
+      pkg.dependencies[depName] = `^${version}`
+    }
+  }
+  if (pkg.devDependencies) {
+    for (const depName of Object.keys(pkg.devDependencies).filter((x) =>
+      x.startsWith('@redwoodjs/')
+    )) {
+      console.log(
+        ` - ${depName}: ${pkg.devDependencies[depName]} => ^${version}`
+      )
+      pkg.devDependencies[depName] = `^${version}`
+    }
+  }
+
+  fs.writeFileSync(
+    path.join(pkgPath, 'package.json'),
+    JSON.stringify(pkg, undefined, 2)
+  )
+}
+
+const cwd = path.join(__dirname, '../')
+const cmd = [
+  'yarn lerna version',
+  version,
+  '--force-publish',
+  '--no-push',
+  '--no-git-tag-version',
```

Comment:
Ah, right. We don't want to tag and push until _after_ the Template packages are updated. What about including those final steps as a part of this command as well? If so, maybe add an initial confirmation with input to proceed, e.g. `This will bump packages from [current-version] --> [new-version] and push a version tagged release. Do you want to proceed?`

Reply:
I was thinking that we can break it into two steps the first step to bump the package versions and the second to actually publish them.

Reply:
You know if you follow this path you'll end up implementing a branch management strategy 😆

Reply:
Maybe we can just up a tag `ship-v0.25.0` which does all of the above, tags the subsequent version bumps, and then deletes the `ship-v0.25.0` tag?